perf/reader.go

```diff
@@ -69,14 +73,24 @@ func readRecord(rd io.Reader, rec *Record, buf []byte) error {
 		internal.NativeEndian.Uint16(buf[6:8]),
 	}
 
+	if rawEvents {
```

Comment:
Wouldn't it be nice to check for `header.Type == PERF_RECORD_LOST` and emit `LostSamples` even with raw events?

Reply:
yea, the separate header should make it easier.. not sure yet if I prefer raw data or the PerfEventHeader,
will check with the rest of the tetragon code first ;-)

same for lost events.. with this new API we actually parse LOST events in upper layer.. I will see how it will look like
to have also LOST events provided, but I'd think it's more straightforward (in already rough API) not to have any exceptions and just return raw events

Reply:
> not to have any exceptions and just return raw events

Yeah I'm coming round to this idea.